packages/components/src/components/Flex/Flex.tsx

```diff
@@ -82,6 +130,9 @@ export type FlexProps = AllowedFrameProps & {
     flex?: Flex;
     flexWrap?: FlexWrap;
     isReversed?: boolean;
+    hasDivider?: boolean;
+    /** @deprecated Use only is case of absolute desperation. Prefer keep it according to elevation. */
```

Comment:
Why adding a prop if we immediately say it's deprecated?

Reply:
Good question, we might delete it if it's not necessary